tests/test_parser.py

```diff
@@ -84,44 +85,65 @@ def test_constructor_cache(self, tmpdir):
         assert df2.cached_content
 
     def test_dockerfile_structure(self, dfparser):
-        dfparser.lines = ["# comment\n",        # should be ignored
-                          " From  \\\n",        # mixed-case
-                          "   base\n",          # extra ws, continuation line
-                          " # comment\n",
-                          " label  foo  \\\n",  # extra ws
-                          "    # comment\n",    # should be ignored
-                          "    bar  \n",        # extra ws, continuation line
+        dfparser.lines = ["# comment\n",                # single-line comment
+                          " From  \\\n",                # mixed-case
```

Comment:
IMHO we should also tests case
```
FROM \
   # comment
   fedora:30
```

Reply:
It's already there (see just below "interrupt instruction"), though ATM it is 2 test cases in one (multi-line comment + comment interrupting a multi-line instruction)

Reply:
It's already there (see just below "interrupt instruction"), though ATM it is 2 test cases in one (multi-line comment + comment interrupting a multi-line instruction)

Reply:
Ahh, sorry for the noise I haven't noticed that